app/src/main/java/de/danoeh/antennapod/fragment/AllEpisodesFragment.java

```diff
@@ -334,7 +334,7 @@ View onCreateViewHelper(LayoutInflater inflater,
             onFragmentLoaded();
         }
 
-        emptyView = (View) root.findViewById(R.id.emptyView);
+        emptyView = root.findViewById(R.id.emptyView);
```

Comment:
Please do not add unrelated changes as this makes the git history harder to understand.

Reply:
Thanks for the feedback. The changes to remove the compiler warnings are already in a separate commit (fd2424), but I can move them to a separate PR if it makes it easier to understand.